linkerd/app/outbound/src/http/logical.rs

```diff
@@ -100,11 +122,9 @@ impl<E> Outbound<E> {
                             .push(http::BoxRequest::layer())
                             .push(http::BoxResponse::layer()),
                     )
-                    .push_map_target(|logical: Logical| {
-                        R::Endpoint::from((
-                            tls::NoClientTls::NotProvidedByServiceDiscovery,
-                            logical,
-                        ))
+                    .push_map_target({
+                        let no_tls_reason = no_tls_reason;
```

Comment:
what is this doing, exactly? is this a copy?

Reply:
forgot we didn't need to rebind `Copy`